File: C/ex28.c
//Exibir a tabuada dos valores de um a vinte, no intervalo de um a dez. Entre as tabuadas, 
//solicitar que o usuário pressione uma tecla.

#include <stdio.h>
#include <stdlib.h>
#include <conio.h>

int main(int argc, char const *argv[])
{
    // Declaração de variáveis
    int r, x;
x=1;
while(20>=x){
for (int i=0; i <= 10; i++){
    r = x * i;
    printf("%i x %i = %i\n", x, i, r);
  }
  getch();
  x++;
}

    return 0;
}

File: Python/ex28.py
#Exibir a tabuada dos valores de um a vinte, no intervalo de um a dez. Entre as tabuadas, 
# solicitar que o usuário pressione uma tecla.


num = 1

for k in range(1, 21, 1):
    for i in range (1, 11, 1):
        tabuada = num*i
        print(f"{num} X {i} = {tabuada}")
    num = num+1
    if k == 20:
        break
    input("\nPara iniciar a proxima tabuada digite Enter\n")
    continue